about.html

```diff
@@ -0,0 +1,131 @@
+                  </ul>
+                </div>
+              </nav>
+
+
+          </header>
+
+      <section class="frame4">
+        <div class="heroContent">
+            <div class="aboutContentGroup">
+                <div class="theme">
+                    <h3 id="themeSub">"Hello! Adventurer."</h6>
+                    <h1 id="themeTitle">NPConnect - What's it all about?
+                    </h1>
+                </div>
+                <div class="aboutText">
+                    <h4>Welcome to D&D Hub, the ultimate online destination for Dungeons and Dragons enthusiasts! Our platform is dedicated to providing a seamless and immersive gaming experience for players and Dungeon Masters (DMs) alike. Whether you're a seasoned adventurer or a newcomer to the realms of tabletop role-playing, D&D Hub is here to support your journey.
+                    </h4>
+                </div>
+            </div>
+            <div class="frame1">
+                <h4>Please contact us per Email for any further questions about NpConnect</h3>
+                    <a href="#"><p>NPConnect@Connectinggamers.com</p></a>
+            </div>
+        </div>
+    </section>
+
+    <section class="logoSection">
+        <div class="frame3">
+            <div class="frame2">
+                <div class="frame1">
+                    <h2>NPConnect Logo</h2>
+                    <div id="ruler"></div>
+                </div>
+                <p>The logo of NPConnect decided by the founder. The logo with the shield and battle axes represents the brother and sisterhood of all players. We protect eachoter. We fight for eachother. We are a party!!</p>
+            </div>
+                <img class="logoSectionimage" src="./assets/logo/svg/Color logo with background.svg">
+        </div>
+    </section>
+    <hr>
+    <section class="previousGatherings">
+        <div class="frame3">
+            <div class="frame2">
+                <div class="frame1">
+                    <h2 class="">NPConnect past events and costumes</h2>
+                    <div id="ruler"></div>
+                </div>
+                <p>"As I stare into the crystallball I see past."</p>
+            </div>
+        </div>
+        <div class="previousGatheringsImage">
+            <div class="previousImgContain">
+                <img class="imageOver" src="./assets/previous events/pexels-cristhian-covaleda-922909.jpg">
+                <div class="overlay">
+                    <h2>2022</h2>
+                    <p>Tournament</p>
+                </div>
+            </div>
+            <div class="previousImgContain">
+                <img class="imageOver" src="./assets/previous events/pexels-pixabay-35797.jpg">
+                <div class="overlay">
+                    <h2>2022</h2>
+                    <p>Top costumes</p>
+                </div>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] Finally, the bottom section should also match our given design and should have the following:
- a heading
- a line of some text
- two divs of images with text in the middle of each message

Given design
![image](https://github.com/George7h/Module-1-capstone-project/assets/81932478/1e418280-df30-4ae9-a423-0d80b7c207da)